src/build.py

```diff
@@ -937,6 +936,16 @@ def LLVM():
   proc.check_call(['ninja', '-v'] + jobs, cwd=LLVM_OUT_DIR, env=cc_env)
   proc.check_call(['ninja', 'install'] + jobs, cwd=LLVM_OUT_DIR, env=cc_env)
   CopyLLVMTools(LLVM_OUT_DIR)
+  install_bin = os.path.join(INSTALL_DIR, 'bin')
+  for target in ('clang', 'clang++'):
+    link = os.path.join(install_bin, 'wasm32-' + target)
+    try:
```

Comment:
Why not just say IsWindows here?

Reply:
Done; yeah this logic was leftover from a different attempt.